### DISCUSSION TITLE
chapter09andhalf/webapp/ScrapeWithPandas，line101: records== gazpacho_records  ,False

### DISCUSSION THREAD

**Discussion by lpf1573** [headfirstpython/third#7] [Q&A] [answered]
when I run the code in jupter:
records== gazpacho_records
the answer is False ,I don't know why .   HELP!

**Comment by barryp-work** [comment, marked as answer]
Hi. You asked this previously, and I answered it here: [https://github.com/headfirstpython/third/discussions/6#discussioncomment-10434189](url) - you need to make sure your data file has the updated results values from the web.  

[Update: not sure why that embedded link doesn't want to work, but a copy'n'paste on it does the trick].

**Reply by lpf1573** [reply]
Thanks ,I finished biting <head first python, 3rd>.This book is my first book that teach me how to code,the way how to learn python keep in my mind. Respect_!

**Reply by headfirstpython** [reply]
Thanks for the kind words.  Wishing you all the best of luck with your future programming challenges. --Paul.
